ui/src/shared/components/RefreshingGraph.tsx

```diff
@@ -133,7 +133,10 @@ class RefreshingGraph extends Component<Props> {
       grabDataForDownload,
     } = this.props
 
-    if (!queries.length) {
+    if (
+      !queries.length ||
+      (queries[0].type === 'flux' && !_.get(queries, '0.text', '').trim())
```

Comment:
it could be nice to extract this to a variable, e.g. `const isEmptyFluxQuery = queries[0].type === `..., since it's not too clear what this guard is doing without the context of the pull request

Reply:
sounds good. can do